Gems/AtomLyIntegration/AtomBridge/Code/Source/FlyCameraInputComponent.cpp

```diff
@@ -300,6 +302,16 @@ void FlyCameraInputComponent::OnKeyboardEvent(const InputChannel& inputChannel)
     {
         m_movement.x = inputChannel.GetValue();
     }
+
+	if(channelId == InputDeviceKeyboard::Key::AlphanumericE)
+	{
+		m_movement.z = inputChannel.GetValue();
+	}
+
+	if(channelId == InputDeviceKeyboard::Key::AlphanumericQ)
+	{
+		m_movement.z = -inputChannel.GetValue();
+	}
```

Comment:
Also tabs have snuck in to this file and causes the whitespace validator to fail.
```suggestion
    if(channelId == InputDeviceKeyboard::Key::AlphanumericE)
    {
        m_movement.z = inputChannel.GetValue();
    }

    if(channelId == InputDeviceKeyboard::Key::AlphanumericQ)
    {
        m_movement.z = -inputChannel.GetValue();
    }
```

Reply:
Done.